package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "paybutton",
+  "version": "2.1.2",
+  "description": "The easiest way to accept eCash online",
+  "author": "poldridge",
+  "main": "index.js",
+  "repository": "https://github.com/PayButton/paybutton.git",
+  "license": "MIT",
+  "scripts": {
+    "build:react":"cd react && yarn && yarn build",
```

Comment:
Could you add `|| cd ..` to all these build commands that `cd` into somewhere? This will make it go back in case any command fails (yarn build or yarn)

Reply:
if the script fails for some reason it will come back to the root path automatically

Reply:
Indeed, I forgot it runs in a subshell so no need to worry about that anymore